### ISSUE TITLE
Problem about the testbench

### ISSUE THREAD

**Issue by NC-Lin-bo** [nvdla/hw#137]
Hello,
I have a question about the testbench (directory names cc_alexnet_conv5_relu5_int16_dtest_cvsram) included three files, input.txn as Alexnet description , input_feature_map.dat as input data, input_weight.dat as weights in neural network.

1. I want to know whether the Alexnet description file in the testbench is complete or not?
2. If the answer is not, so I guess the alexnet only run a specific layers, maybe only one convolution layer executed on NVDLA hardware?
3. How can I modified the NVDLA hardware or even the runtime execution to let it executes the whole, complete Alexnet?

Thank you so much!

**Comment by silvaurus** [comment]
See https://github.com/nvdla/sw

**Comment by NC-Lin-bo** [comment]
Thanks, @silvaurus.
I want to say could I generate testbench of complete Alexnet for NVDLA hardware.

**Comment by silvaurus** [comment]
Yeah I think from what I understand you need to have a virtual environment and NVDLA software support (real time environment) to compile your NN implementation then generate hardware testbench.

**Comment by NC-Lin-bo** [comment]
Thank, @silvaurus 
I know virtual platform, but I want to detail timing information by RTL.

**Comment by ghost** [comment]
> Hello,
> I have a question about the testbench (directory names cc_alexnet_conv5_relu5_int16_dtest_cvsram) included three files, input.txn as Alexnet description , input_feature_map.dat as input data, input_weight.dat as weights in neural network.
> 
> 1. I want to know whether the Alexnet description file in the testbench is complete or not?
> 2. If the answer is not, so I guess the alexnet only run a specific layers, maybe only one convolution layer executed on NVDLA hardware?
> 3. How can I modified the NVDLA hardware or even the runtime execution to let it executes the whole, complete Alexnet?
> 
> Thank you so much!

hello,
once I generated the hardware description file using vivado, how to control my nvdla through software environment and how to create software environment .please help me out
